### ISSUE TITLE
PCA flip sign

### ISSUE THREAD

**Issue by hainm** [Amber-MD/cpptraj#87]
In the process of satisfying my curiosity, I compare PCA between cpptraj and sklearn.

Both give the same absolute result for the projection values. However, the sign is opposite. 

Please check the plot here
![pca_cpptraj](https://cloud.githubusercontent.com/assets/4451957/10124628/4ecb68aa-6527-11e5-8b8b-20d5a6e4bab9.png)

![pca](https://cloud.githubusercontent.com/assets/4451957/10124622/3506e7f0-6527-11e5-81db-75b600919de9.png)

is it your intention? (already tried to search `PCA flip sign cpptraj` on google)

**Comment by drroe** [comment]
@hainm this has to do with the fact that eigenvector "sign" is not necessarily meaningful. If you have an eigenvector `x` that satisfies
```
A * x = L * x
```
(where A is your square matrix and L is the eigenvalue) then `-x` also satisfies it, i.e. if `x` is an eigenvector then `-x` is as well. In practice, the "sign" of an eigenvector depends on the math library used to calculate it. This is in fact why the `modes` command displacement test is disabled for cpptraj. I bet if you use a different math library (e.g. use your system BLAS/LAPACK instead of the one bundled with Amber or vice versa) you will get different signs.

Bottom line is that eigenvector sign doesn't matter.
